### PR TITLE
feat(useWakeLock): new function

### PR DESCRIPTION
**Pull request by koheing**
Hi, 
Thank you for great library!

This PR is the implementation of #667 .

### CODE REVIEW

**Comment by wheatjs** [comment]
Looks good to me 👍

**Comment by koheing** [comment]
I'm sorry. I forgot to add to markdown and index.json, so I pushed commit again.

**Comment by wheatjs** [comment]
Those files will be autogenerated so you don't need to worry about manually adding them

**Comment by koheing** [comment]
Thanks for letting me know!
I read contributing guidelines, but I missed that.

File: packages/core/useWakeLock/index.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  async function onVisibilityChange() {
+    if (!isSupported || !wakeLock) return
+
+    if (document && document.visibilityState === 'visible')
+      wakeLock = await (navigator as NavigatorWithWakeLock).wakeLock.request('screen')
+
+    isActive.value = !wakeLock.released
+  }
+
+  useEventListener(document, 'visibilitychange', onVisibilityChange, { passive: true })
```
Comment (antfu) [inline packages/core/useWakeLock/index.ts:37]: We will need to have `document` configurable as well.

**Review by antfu — CHANGES_REQUESTED** [review]

File: packages/core/useWakeLock/index.ts (hunk 1 above)
Comment (koheing) [reply packages/core/useWakeLock/index.ts]: Thanks for letting me know!
I fixed it, so could you please confirm it again?

**Review by antfu — APPROVED** [review]

**Pull request merged**